spec/app/ics-027-interchain-accounts/README.md

```diff
@@ -57,9 +57,13 @@ call 04-channel `ChanOpenInit`. An error is returned if the controller portID is
 A `ChannelOpenInit` event is emitted which can be picked up by an offchain process such as a relayer.
 The account will be registered during the `OnChanOpenTry` step on the host chain.
 This function must be called after an `OPEN` connection is already established with the given connection identifier.
+The caller must provide the complete version. This MUST include the ICA version with complete metadata and it MAY include 
```

Comment:
```suggestion
The caller must provide the complete channel version. This MUST include the ICA version with complete metadata and it MAY include 
```

?